faasmcli/faasmcli/tasks/codegen.py

```diff
@@ -93,8 +93,27 @@ def local(ctx):
         _do_codegen_file(so)
 
     # Run the WAMR codegen required by the tests
-    codegen(ctx, "demo", "echo", wamr=True)
+    # Misc
     codegen(ctx, "demo", "chain", wamr=True)
+    codegen(ctx, "ffmpeg", "check", wamr=True)
+
+    # Environment tests
+    codegen(ctx, "demo", "argc_argv_test", wamr=True)
+    codegen(ctx, "demo", "exit", wamr=True)
+    codegen(ctx, "demo", "getenv", wamr=True)
+
+    # Filesystem
+    codegen(ctx, "demo", "fcntl", wamr=True)
+    codegen(ctx, "demo", "file", wamr=True)
+    codegen(ctx, "demo", "filedescriptor", wamr=True)
+    codegen(ctx, "demo", "fstat", wamr=True)
+    codegen(ctx, "demo", "fread", wamr=True)  # not working
```

Comment:
Hopefully I can upstream the changes to get this to work: bytecodealliance/wasm-micro-runtime#766